site/docs/v1/tech/installation-guide/kickstart.adoc

```diff
@@ -251,7 +248,7 @@ If you don't create a tenant using the Tenant API in your kickstart file then yo
 
 There is a top level property in the request called `tenantId` and you simply set that value to indicate which Tenant you wish to use.
 
-In this example, we'll create a new application and we'll do it in a second tenant that we create. Because I need to know the `tenantId` I am generating a new UUID using the `#{UUID()}` function call and assigning that value to `secondTenantId`. Now I can re-use this value to create the tenant, and to make the Create Application API request.
+Create a new application in a second, new tenant. Because you need to know the `tenantId`, generate a new UUID using the `#{UUID()}` function call and assign that value to `secondTenantId`. Now you can re-use this value to create the tenant, and to make the Create Application API request.
```

Comment:
```suggestion
Create a new application in a second, new tenant. Because you need to know the `tenantId`, generate a new UUID using the `#{UUID()}` function call and assign that value to `secondTenantId`. Now, you can re-use this value to create the tenant and to make the Create Application API request.
```